places-solution-integrations/analytics/places-trigger.md

```diff
@@ -0,0 +1,68 @@
+# Using Places Events to Send Data to Analytics via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
+
+Once Places is sending in events for entries and exits, you can leverage Rules in Launch to send data to Adobe Analytics. With your desired property selected in Launch, you can create this rule by completing four steps:
```

Comment:
"After Places sends the entry and exit events, you can create rules in Experience Platform Launch and attach your Places data to all Adobe Analytics events. To create this type of rule, select your property in Launch and complete the following steps: